app/models/mongo_fulfillment.rb

```diff
@@ -0,0 +1,36 @@
+# Indicator that a pledge by the primary fund was cashed in,
+# which in turn indicates that the patient used our pledged money.
+class Fulfillment
+  include Mongoid::Document
+  include Mongoid::Timestamps
+  include Mongoid::History::Trackable
+  include Mongoid::Userstamp
+
+  # Relationships
+  embedded_in :can_fulfill, polymorphic: true
```

Comment:
All of our previous ports added a line like:

```
  store_in collection: 'patients'
```

I'm assuming this preexisting relationship is why we don't need this here?

Reply:
exactly right - the document store knows to save this as a subobj within patient objects in the appropriate collection, so no need to declare.